internal/scorecard/plugins/olm_tests.go

```diff
@@ -165,9 +170,9 @@ func NewOLMTestSuite(conf OLMTestConfig) *schelpers.TestSuite {
 		"Test suite checks if an operator's CSV follows best practices",
 	)
 
+	ts.AddTest(NewBundleValidationTest(conf), 1)
 	ts.AddTest(NewCRDsHaveValidationTest(conf), 1.25)
 	ts.AddTest(NewCRDsHaveResourcesTest(conf), 1)
-	ts.AddTest(NewAnnotationsContainExamplesTest(conf), 1)
 	ts.AddTest(NewSpecDescriptorsTest(conf), 1)
```

Comment:
Same ^

Reply:
same as above, this test was removed since csv validation is done before any tests are run with changes in this PR.  The CSV validation by the API includes the examples check.